android/filament-android/src/main/java/com/google/android/filament/LightManager.java

```diff
@@ -240,7 +240,7 @@ public static class ShadowOptions {
          */
         @NonNull
         @Size(min = 3)
-        public float[] cascadeSplitPositions = { 0.25f, 0.50f, 0.75f };
+        public float[] cascadeSplitPositions = { 0.125f, 0.25f, 0.50f };
```

Comment:
0.125f as the first split might be too close. In debug view I often need to move the first split farther away because with the default fov it’s outside of the view frustum often.

Reply:
It is quite close but still visible on our demos (e.g. sponza). It also works a lot better for "stable" mode, and provides a visually linear partitioning. In stable mode with the default far plane, the first split is at about 12m, it's pretty far. With the old default it would be 25m and so it looks like it's not even working.